RevenueCatUI.podspec

```
@@ -16,9 +16,12 @@ Pod::Spec.new do |s|
   s.framework      = 'SwiftUI'
   s.swift_version  = '5.7'
 
-  # Technically PaywallView isn't available until iOS 15,
-  # but this can be detected at compile time.
+  # Technically PaywallView isn't available in all these platforms / versions but this can be detected at compile time.
```

the language here makes it seem like a hack, but it is actually a reasonable way of managing this. Perhaps we can make this clearer? 
```suggestion
  # RevenueCatUI APIs are not available in all these platforms / versions, however retaining this support at the Pod level 
  # allows us to depend on it in the same platforms as RevenueCat.
```